lib/logger/src/main/scala/org.bitbucket.pshirshov.izumi/config/ConfigLoader.scala

```diff
@@ -0,0 +1,9 @@
+package org.bitbucket.pshirshov.izumi.config
+
+import org.bitbucket.pshirshov.izumi.logger.Log
+import org.bitbucket.pshirshov.izumi.logger.Models.{LoggingConfig, LoggingRule}
+
+trait ConfigLoader {
+
+  lazy val loggingConfig: LoggingConfig = LoggingConfig(rules = Seq(LoggingRule("", Set(Log.Level.Debug))))
```

Comment:
You shouldn't think about logger configs at this stage, you should concentrate on API. Also `lazy val` is not a good thing.